docs/resources/resource.md

```diff
@@ -22,16 +22,20 @@ resource "twingate_remote_network" "aws_network" {
   name = "aws_remote_network"
 }
 
+resource "twingate_group" "aws" {
+  name = "aws_group"
+}
+
 resource "twingate_resource" "resource" {
   name = "network"
   address = "internal.int"
   remote_network_id = twingate_remote_network.aws_network.id
-  group_ids = ["group1"]
+  group_ids = [twingate_group.aws.id]
   protocols {
     allow_icmp = true
     tcp  {
-      policy = "RESTRICTED"
-      ports = ["80", "82-83"]
+      policy = "DENY_ALL"
+      ports = ["80", "82-83", "90"]
```

Comment:
What does it mean when you `DENY_ALL` but also give `ports` ?
This should error... or warn?

Reply:
ports will be ignored in this case

Reply:
yeah, this was not specified

Reply:
So we shouldn't give that as an example :). you should probably change it back to `RESTRICTED`

Reply:
ok, will change to `RESTRICTED`

and in case user provides non-empty list of ports with DENY_ALL, what error message should be?

Reply:
@ekampf changed back to `RESTRICTED`